plugins/lookup/nb_lookup.py

```diff
@@ -256,6 +256,26 @@ def get_endpoint(netbox, term):
         "webhooks": {"endpoint": netbox.extras.webhooks},
     }
 
+    major, minor, patch = map(int, pynetbox.__version__.split("."))
+
+    if major >= 6 and minor >= 4 and patch >= 0:
+        netbox_endpoint_map["wireless-lan-groups"] = {
+            "endpoint": netbox.wireless.wireless_lan_groups
+        }
+        netbox_endpoint_map["wireless-lan-groups"] = {
+            "endpoint": netbox.wireless.wireless_lan_groups
+        }
+        netbox_endpoint_map["wireless-lan"] = {"endpoint": netbox.wireless.wireless_lan}
+        netbox_endpoint_map["wireless-links"] = {
+            "endpoint": netbox.wireless.wireless_links
+        }
+
+    else:
+        Display().v(
```

Comment:
`-v` is going to end up printing this in everyone's playbooks. This is not the right verbosity level.

Reply:
Please advise on what the right verbosity level you believe it should be.  I've changed it to only complain when you try to use the wireless-* lookups.

Reply:
That sounds good to me